packages/router/src/apply_redirects.ts

```diff
@@ -96,6 +96,7 @@ class ApplyRedirects {
     return urlTrees$.pipe(catchError((e: any) => {
       if (e instanceof AbsoluteRedirect) {
         // after an absolute redirect we do not apply any more redirects!
```

Comment:
nit (for consistency): 
```suggestion
        // After an absolute redirect we do not apply any more redirects!
```